scripts/covid_owid_project/preprocess_csv.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import io
+import ssl
+import urllib.request
+
+output_columns = [
+    'Date', 'GeoId',
+    'CumulativeCount_Vaccine_COVID_19_Administered',
+    'IncrementalCount_Vaccine_COVID_19_Administered',
+    'CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedCase',
+    'IncrementalCount_MedicalConditionIncident_COVID_19_ConfirmedCase',
+    'CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased',
+    'IncrementalCount_MedicalConditionIncident_COVID_19_PatientDeceased',
+    'Count_MedicalConditionIncident_COVID_19_PatientInICU',
+    'Count_MedicalConditionIncident_COVID_19_PatientHospitalized',
+    'CumulativeCount_MedicalTest_ConditionCOVID_19',
+    'IncrementalCount_MedicalTest_ConditionCOVID_19'
+]
+with open('COVID_OWID.csv', 'w', newline='') as f_out:
+    writer = csv.DictWriter(f_out,
+                            fieldnames=output_columns,
+                            lineterminator='\n')
+    gcontext = ssl.SSLContext()                         
+    with urllib.request.urlopen(
+            'https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/owid-covid-data.csv', context=gcontext) as response:
+        reader = csv.DictReader(io.TextIOWrapper(response))
+
+        writer.writeheader()
+        for row_dict in reader:
+            # Skip invalid country ISO code.
+            if len(row_dict['iso_code']) != 3 : 
```

Comment:
Might be good to restrict it to the ones listed here: https://github.com/datacommonsorg/data/blob/master/util/alpha2_to_dcid.py#L16

Unless you've confirmed there aren't invalid 3-letter country codes?